spec/lib/evm_database_ops_spec.rb

```diff
@@ -8,7 +8,7 @@
       allow(MiqGenericMountSession).to receive(:new_session).and_return(session)
       allow(session).to receive(:settings_mount_point).and_return(Rails.root.join("tmp").to_s)
       allow(session).to receive(:uri_to_local_path).and_return(Rails.root.join("tmp/share").to_s)
-      allow(PostgresAdmin).to receive(:runcmd_with_logging)
+      allow(PostgresAdmin).to receive(:backup_pg_compress).and_return("/tmp/backup_1")
```

Comment:
At this point couldn't we just stub the public interface (`backup`)?

Reply:
Sure.  I was under the impression we still had the:

```ruby
if doing_a_dump?
  backup_pg_dump
else
  backup_pg_compress
end
```

But I guess I was wrong... so I agree.  Will switch.